### ISSUE TITLE
Website shifting for Firefox

### ISSUE THREAD

**Issue by woj-tek** [thompsonate/Shifty#116]
I know that currently only Safari and Chromium-based browsers are supported but it would be awesome if, in the future, Firefox would also gain support.

(what are the requirements to get it working? I assume you use some sort of API to get the information about current URL, which is provided by Chromium-based browsers?)

**Comment by choco** [comment]
Yes, you're correct. For Chromium based browsers we use `ScriptingAdditions` api, for Safari we use either that or the `Accessibility` api. Unfortunately neither of them are supported by Firefox. If either of them are added/exposed in the future we'll definitely add support for Firefox!
A more complex way would be to create a Firefox extension that uses the native messaging api to connect to shifty and send current tab url.

**Comment by ridicolos** [comment]
Would love to see this be implemented :)
